app/helpers/hera_helper.rb

```diff
@@ -26,34 +50,18 @@ def flash_messages
     end.join("\n").html_safe
   end
 
+  def in_project?
+    defined?(current_project) && current_project&.persisted?
```

Comment:
```suggestion
    current_project && current_project.persisted?
```

Reply:
we don't need the safeguard since we shouldn't end up on the right side of this expression unless `current_project` exists

Reply:
Unless this is due to a strange case where it's defined and not set? There is probably a nicer way to handle this case